spark/src/test/scala/org/apache/comet/exec/CometAggregateSuite.scala

```diff
@@ -974,7 +974,8 @@ class CometAggregateSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
-  test("distinct") {
+  // TODO enable once https://github.com/apache/datafusion-comet/issues/1267 is implemented
```

Comment:
so now this tests passed?

Reply:
I'm thinking should we add a test with unique values, that fails now by correctness issue

Reply:
yes, many of the tests were passing because the aggregate was running in Spark rather than Comet or because the input values were already distinct

Reply:
I'm looking at this test again on main and do not understand how this one is passing now :confused:

Reply:
I understand what is happening now. Spark optimizes some simple cases like `SELECT a, COUNT(DISTINCT b) ...` to first perform an aggregate to just get the distinct values, so by the time we perform the `COUNT` there are only distinct values.